planetarium/NineChronicles
# Introduce `UnlockWorld` action with UI

### Description

1. SSIA

### How to test

1. Clear last stage of world.
2. Check next world locking.
3. Try to unlock worlds.

### Related Links

[월드 오픈 액션 9c에 반영](https://app.asana.com/0/958521740385861/1202257909821553/f)
[월드 오픈 UI&연출 플로우 적용](https://app.asana.com/0/958521740385861/1202257909821558/f)

### Required Reviewers

@Unengine @BBeom06 @jaeho0103 

### Screenshot
![image](https://user-images.githubusercontent.com/48484989/168745834-70b2b890-ab7c-41fd-850d-a81862b9fd38.png)
![image](https://user-images.githubusercontent.com/48484989/168745872-b84d3d18-b8b4-4ed7-af07-266e421ff47c.png)
![image](https://user-images.githubusercontent.com/48484989/168751071-e50b887e-6d5b-416a-843e-1e13b4510dff.png)
